modules/m_away.c

```diff
@@ -89,6 +90,10 @@ m_away(struct MsgBuf *msgbuf_p, struct Client *client_p, struct Client *source_p
 
 			sendto_common_channels_local_butone(source_p, CLICAP_AWAY_NOTIFY, NOCAPS, ":%s!%s@%s AWAY",
```

Comment:
Exclude extended-monitor

```suggestion
			sendto_common_channels_local_butone(source_p, CLICAP_AWAY_NOTIFY, CLICAP_EXTENDED_MONITOR, ":%s!%s@%s AWAY",
```